watcher.go

```diff
@@ -175,3 +184,34 @@ func updatePromSDConfigMap(client *kubernetes.Clientset, fileSDConfig *v1.Config
 	}
 	return nil
 }
+
+func checkDuplicates(values []map[string][]string, value string) bool {
+	for _, target := range values {
+		for _, ip := range target["targets"] {
+			if ip == value {
+				return true
+			}
+		}
+	}
+	return false
+}
+
+func deDupStaticConfig(values []map[string][]string) []map[string][]string {
```

Comment:
nit: this deduplicates any configuration right?  should we just name this `dedupConfig` ?